public/components/TagEdit/formComponents/TagImageEdit.react.js

```diff
@@ -111,7 +110,7 @@ export default class TagImageEdit extends React.Component {
     }
 
     if (this.state.metadataFetchStatus === FETCH_STATES.success) {
-      return (<div className="tag-edit__image__add--success" onClick={this.addImage.bind(this)}><i className="i-tick-green" />Add Image</div>);
+      return (<div className="tag-edit__image__add--success" onClick={this.addImage.bind(this)}><i className="i-tick-green" />Click here to add the image</div>);
```

Comment:
Expanded the text here because some newer users to tag manager weren't aware that you had to click the `Add Image` text to add the image. Which is fair enough since it doesn't look like a button at all - it just looks like a notification that your image has been added.